modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/CustomElementCETImplFactoryImpl.java

```diff
@@ -81,14 +84,18 @@ public void validate(
 
 		if (!matcher.matches()) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"please-enter-a-valid-friendly-url-mapping");
+				"Invalid friendly URL mapping",
+				"please-enter-a-valid-friendly-url-mapping"
+			);
 		}
 
 		String htmlElementName = newCustomElementCET.getHTMLElementName();
 
 		if (Validator.isNull(htmlElementName)) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"html-element-name-is-empty");
+				"Custom element HTML element name is null",
```

Comment:
I think we can keep this as just `HTML element` instead of `Custom element HTML element` to match the language key. Please change this for all instances.